mandatory/2-function-creation.js

```diff
@@ -16,7 +21,13 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (num >= 100) {
+    return "100%";
+  } else {
+    return `${num.toFixed()}%`;
```

Comment:
Hi, I am reviewing your code and you have done great! Looking at your code helped me spot a mistake in mine, so thanks. Here I think your code does not round the numbers to 2 decimals, you could use `Math.round` here, it helped me pass the test.

Reply:
Hi Timea, thanks for the review. If you look at the updated mandatory exercise .toFixed(2) rounds it up to 2 decimal places. I believe Math.round only rounds it up to the nearest integer. please correct me if i'm wrong.